app/components/ViewContainer.js

```diff
@@ -8,7 +8,12 @@ export default class ViewContainer extends Component {
     super(props);
     this.state = {
       view: 'map',
-      currLoc: null,
+      currLoc: {
+        latitude: 37.78825,
+        longitude: -122.4324,
+        latitudeDelta: 0.0922,
+        longitudeDelta: 0.0421,
```

Comment:
What are the latitudeDelta and longitudeDelta for in the currLoc? The region code in Map doesn't seem to make use of props.currLoc for the deltas